jupyter_client/manager.py

```diff
@@ -488,6 +509,221 @@ def is_alive(self):
             return False
 
 
+class AsyncKernelManager(KernelManager):
+    """Manages kernels in an asynchronous manner """
+
+    client_factory = Type(klass='jupyter_client.asynchronous.AsyncKernelClient')
+
+    async def _launch_kernel(self, kernel_cmd, **kw):
+        """actually launch the kernel
+
+        override in a subclass to launch kernel subprocesses differently
+        """
+        res = launch_kernel(kernel_cmd, **kw)
+        return res
```

Comment:
There is nothing `async` about this function, it doesn't `await` anything.

Reply:
Thanks for the review David, but I don't agree with this comment (and the others below) regarding the marking of a method with `async` that doesn't `await`.  While it's true that _this implementation_ doesn't await.  Subclasses, especially those managing remote kernels, certainly will, since the times to start, interrupt, signal, etc. take an order of magnitude longer (perhaps more) to complete.  

Because callers won't know they're talking to a kernel manager that is dealing with remote kernels (nor should they), I believe the async markers need to remain so the caller is always performing `await start_kernel()`, for example.  

Similarly, I think `is_alive()` needs to be added back into AsyncKernelManager since `poll()` is an RPC in some remote kernels (and a REST call in others).  And, by extension, I believe AsyncKernelClient needs an async `is_alive()` method since it will call on the kernel manager's `is_alive()` and we'll need to check the class to conditionally apply the `await`.  I have such an incantation working right now, but I wanted to get your opinion before its push.

Reply:
You convinced me @kevin-bates! Thanks for putting this work into perspective, I had not thought of the remote kernel use case. It makes more sense now, so please discard my previous comments.